libs/coin-modules/coin-polkadot/src/test/coin-tester/scenarios/basic.ts

```diff
@@ -246,7 +246,7 @@ export const basicScenario: Scenario<PolkadotTransaction, PolkadotAccount> = {
   name: "Polkadot Basic transactions",
   setup: async () => {
     const [{ transport, getOnSpeculosConfirmation }] = await Promise.all([
-      spawnSpeculos(`/${defaultNanoApp.firmware}/Polkadot/app_${defaultNanoApp.version}.elf`),
+      spawnSpeculos(`/${defaultNanoApp.firmware}/PolkadotMigration/app_${defaultNanoApp.version}.elf`),
```

Comment:
Is this the definitive URL ?

Reply:
yes, it is the definitive URL. but they've just publish this app for the firmware 2.3.0.
I will update the firmware version.
See https://github.com/LedgerHQ/coin-apps/tree/master/nanox/2.3.0/PolkadotMigration